src/Temporalio/Workflows/WorkflowQueryDefinition.cs

```diff
@@ -10,11 +11,31 @@ namespace Temporalio.Workflows
     /// </summary>
     public class WorkflowQueryDefinition
     {
+        /// <summary>
+        /// All known reserved query handler prefixes.
+        /// </summary>
+        internal static readonly string[] ReservedQueryHandlerPrefixes =
```

Comment:
```suggestion
        private static readonly string[] ReservedQueryHandlerPrefixes =
```

Doesn't matter much though